core/adapters/index.ts

```diff
@@ -3,6 +3,7 @@ import { Tasks } from "../tasks";
 import { BigQueryAdapter } from "./bigquery";
 import { RedshiftAdapter } from "./redshift";
 import { SnowflakeAdapter } from "./snowflake";
+import { SQLDataWarehouseAdapter } from "./sqldatawarehouse";
```

Comment:
nit: can you change imports where you find them to be fully-qualified? so in this case, `@dataform/core/adapters/sqldatawarehouse`